sample_apps/rso_sample_apps/rust/config/config.yml

```diff
@@ -0,0 +1,13 @@
+server: # server configuration
```

Comment:
I would include a `config.yml.tmpl` file in the repo and add `config.yml` to gitignore. Otherwise people will accidentally leak their client_secrets.

Reply:
@mkawalec What do you think of a rename to `example-config.yml`? The app itself looks for a config.yml in the local directory and the root `.gitignore` does include `config.yml`. 

I'm trying to express to the consumer that this file should only be used as an example, and not as the actual config.